node-bootstrapper/parser/parser_test.go

```diff
@@ -538,13 +501,13 @@ func getNBCInstance(jsonFilePath string) *nbcontractv1.Configuration {
 	return nBCB.GetNodeBootstrapConfig()
 }
 
-func generateTestDataIfRequested(t *testing.T, folder, cseCmd string) {
+func generateTestDataIfRequested(t *testing.T, folder string, cmd *exec.Cmd) {
 	if os.Getenv("GENERATE_TEST_DATA") == "true" {
```

Comment:
Just seeing this so asking, though it's not part of this PR.
Where is this env var set? ADO pipeline?

Reply:
idk. It looks weird for me. I would delete it.